scripts/sct_register_multimodal.py

```diff
@@ -428,10 +472,17 @@ def main(args=None):
 
     # Generate output files
     sct.printv('\nGenerate output files...', verbose)
+    # src_reg
```

Comment:
I would either provide a more meaningful message on line 473 or avoid it all together. The one word comments are noisy and provide no value.